src/components/ui/Menu/context/MenuContext.tsx

```
@@ -0,0 +1,9 @@
+import { createContext } from 'react';
+
+type MenuContextType = {
+
+};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)